src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp

```diff
@@ -88,7 +91,8 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
     static std::unique_ptr<primitive_impl> create(const typed_program_node<PType>& arg, const kernel_impl_params& impl_param) {
         // concat buffer fusing for dynamic shape is adaptively applied at runtime. So we need to build dynamic impl at build time.
         if (impl_param.can_be_optimized() &&
-            !((impl_param.is_type<concatenation>() ||
+            !((impl_param.runtime_skippable() ||
+               impl_param.is_type<concatenation>() ||
```

Comment:
since we have runtime_skippable, we dont need per primitive condition (except crop)

Reply:
Fixed it. thanks